### PR TITLE
sidenav back button update

### PR DESCRIPTION
**Pull request by bipinshashi**
updating back-button in sidenav and adding `All Projects` text
screen shot 2018-07-10 at 7 52 08 pm

Design Link: https://confluence.hq.unity3d.com/display/CSUXG/Sidenav+visual+update

### CODE REVIEW

File: src/components/sidenav/sidenavBackButton/sidenavBackButton.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
-<div class="ump-sidenav--back">
-  <md-icon ng-click="$ctrl.onClick()">
-    arrow_upward
+<div class="ump-sidenav--back" ng-click="$ctrl.onClick()">
+  <md-icon>
+    arrow_back
   </md-icon>
+  <div class="back-text"> All projects</div>
```
Comment (maciej-gurban) [inline src/components/sidenav/sidenavBackButton/sidenavBackButton.html]: There shouldn't be a space between `>` and `A`. This is also a hard-coded string, which should come from e.g. a binding

File: src/components/sidenav/sidenavBackButton/sidenavBackButton.scss [hunk 2]
```diff
@@ -5,14 +5,21 @@ $button-color-focus: #333;
...
+  }
   .material-icons {
     font-size: 16px;
     font-size: inherit;
   }
-}
+  .back-text {
+    margin-left: 12px;
+  }
+}
```
Comment (maciej-gurban) [inline src/components/sidenav/sidenavBackButton/sidenavBackButton.scss]: Missing newline

File: src/components/sidenav/sidenavBackButton/sidenavBackButton.scss [hunk 3]
```diff
@@ -5,14 +5,21 @@ $button-color-focus: #333;
 $button-size: 16px;
 
 .ump-sidenav--back {
-  @include icon-size($button-size);
-
-  color: $button-color;
-  cursor: pointer;
+  display: flex;
+  min-height: 32px;
```
Comment (maciej-gurban) [inline src/components/sidenav/sidenavBackButton/sidenavBackButton.scss]: Is there a reason this is not just `height`?

File: src/components/sidenav/sidenavBackButton/sidenavBackButton.html (hunk 1 above)
Comment (bipinshashi) [reply src/components/sidenav/sidenavBackButton/sidenavBackButton.html]: updated to a binding

**Comment by bipinshashi** [comment]
@maciej-gurban good to merge?

**Comment by maciej-gurban** [comment]
Not on computer, but feel free. Anybody else should be able to press the
approve button

**Comment by bipinshashi** [comment]
get the following error when trying to publish, do i need to register somewhere to publish?
```
npm publish ./build --access public

npm ERR! code ENEEDAUTH
npm ERR! need auth auth required for publishing
npm ERR! need auth You need to authorize this machine using `npm adduser`

**Pull request merged**